### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by anitas3791** [sephiroth74/Material-BottomNavigation#52]
Hello,

Usually Talkback reads the title of menu items, but noticed that isn't happening with the items in the bottom navigation view. Is there anything I should be doing to have the title be the content description or is this a bug/intentional? Visually impaired users won't be able navigate the app so any guidance is appreciated!

Thanks,
Anita

**Comment by sephiroth74** [comment]
it's definitely a bug. I'll look into this!

**Comment by anitas3791** [comment]
Thank you!!
